packages/next/src/client/components/react-dev-overlay/internal/helpers/hydration-error-info.ts

```diff
@@ -17,20 +17,20 @@ export const hydrationErrorState: HydrationErrorState = {}
 
 // https://github.com/facebook/react/blob/main/packages/react-dom/src/__tests__/ReactDOMHydrationDiff-test.js used as a reference
 const htmlTagsWarnings = new Set([
-  'Warning: Cannot render a sync or defer <script> outside the main document without knowing its order. Try adding async="" or moving it into the root <head> tag.%s',
-  'Warning: In HTML, %s cannot be a child of <%s>.%s\nThis will cause a hydration error.%s',
-  'Warning: In HTML, %s cannot be a descendant of <%s>.\nThis will cause a hydration error.%s',
-  'Warning: In HTML, text nodes cannot be a child of <%s>.\nThis will cause a hydration error.',
-  "Warning: In HTML, whitespace text nodes cannot be a child of <%s>. Make sure you don't have any extra whitespace between tags on each line of your source code.\nThis will cause a hydration error.",
-  'Warning: Expected server HTML to contain a matching <%s> in <%s>.%s',
-  'Warning: Did not expect server HTML to contain a <%s> in <%s>.%s',
+  'Cannot render a sync or defer <script> outside the main document without knowing its order. Try adding async="" or moving it into the root <head> tag.%s',
+  'In HTML, %s cannot be a child of <%s>.%s\nThis will cause a hydration error.%s',
+  'In HTML, %s cannot be a descendant of <%s>.\nThis will cause a hydration error.%s',
```

Comment:
Need to double check if these work as well i.e. I'm not sure   if the last placeholder is the component stack and if we still inject it.

Reply:
Thanks